### PR TITLE
Feature/custom accessor

### PR DESCRIPTION
**Pull request by FreakTheMighty**
## Overview
This pull request builds on the work done in #6953 to support custom vertex data as supported in the glTF spec. In the previous work we were able to get accessor index, but didn't implement a method to access the underlying buffer or  to parse it.

This pull request adds the `GetAccessor` method which allows us to get an accessor by index, and also DRYs up the `ConstructMeshPrimitiveAsync` code. Additionally, the various `GltfConversions` methods have been made public to make parsing the buffer easy.

## Verification

We've added a new GLB with both the accessor and underlying buffer data along with a Unit test that covers the getting and parsing of the buffer data.

### CODE REVIEW

File: Assets/MRTK/Tests/PlayModeTests/GltfTests.cs [hunk 1]
```diff
@@ -98,8 +100,32 @@ public IEnumerator TestGltfCustomAttributes()
...
+
+            foreach (var item in intArray)
+            {
+                Assert.AreEqual(10, item);
+            }
+        }
         #endregion
         
-    }
+    }
```
Comment (keveleigh) [inline Assets/MRTK/Tests/PlayModeTests/GltfTests.cs:129]: nit: extra whitespace

```suggestion
    }
```

File: Assets/MRTK/Tests/PlayModeTests/GltfTests.cs [hunk 2]
```diff
@@ -98,8 +100,32 @@ public IEnumerator TestGltfCustomAttributes()
             int temperature = gltfObject.accessors[temperatureIdx].count;
             Assert.AreEqual(100, temperature);
         }
+        
+        [UnityTest]
+        public IEnumerator TestGltfCustomAttributesData() {
+            // Load glTF
```
Comment (keveleigh) [inline Assets/MRTK/Tests/PlayModeTests/GltfTests.cs:106]: Small nit to match brace style in the rest of the file:

```suggestion
        public IEnumerator TestGltfCustomAttributesData()
        {
            // Load glTF
```

File: Assets/MRTK/Core/Utilities/Gltf/Serialization/GltfConversions.cs [hunk 3]
```diff
@@ -7,7 +7,7 @@
 
 namespace Microsoft.MixedReality.Toolkit.Utilities.Gltf.Serialization
 {
-    internal static class GltfConversions
+    public static class GltfConversions
```
Comment (keveleigh) [inline Assets/MRTK/Core/Utilities/Gltf/Serialization/GltfConversions.cs:10]:
```suggestion
    /// <summary>
    /// Provides data accessors and conversions useful while reading and using glTF objects.
    /// </summary>
    public static class GltfConversions
```

**Comment by FreakTheMighty** [comment]
@keveleigh thanks for the review and the style updates!

**Pull request merged**